core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -688,7 +701,7 @@ const worgen = (card: ReferenceCard) => race(card, Race.WORGEN);
 
 const alliance = or(draenei, dwarf, gnome, highelf, human, nightelf, worgen);
 const horde = or(bloodelf, goblin, halforc, orc, tauren, troll, undead);
-const explorer = (card: ReferenceCard) => hasMechanic(card, GameTag.MERCS_EXPLORER);
+const explorer = (card: ReferenceCard) => hasTag(card, GameTag.MERCS_EXPLORER);
```

Comment:
We will need to separate the referencedTags, which are tags that are referenced by the card, from tags that describe the card itself.
In other words, a card that has MERCS_EXPLORER in its referencedTags is a card that acts upon MERCS_EXPLORER cards.

I will introduce a new "tags" field, which will hold this MERCS_EXPLORER tag.